manifests/install.pp

```diff
@@ -10,10 +10,15 @@
   if $telegraf::manage_repo {
     case $facts['os']['family'] {
       'Debian': {
+        if $facts.dig('os','distro','codename') {
+          $release = $facts['os']['distro']['codename']
+        } else {
+          $release = $facts['os']['lsb']['codename']
+        }
         apt::source { 'influxdata':
           comment  => 'Mirror for InfluxData packages',
           location => "${telegraf::repo_location}${facts['os']['name'].downcase}",
-          release  => $facts['os']['distro']['codename'],
+          release  => $release,
```

Comment:
Wasn't it better to just leave this parameter out? https://github.com/puppetlabs/puppetlabs-apt/blob/2df4e59aad25864f7d674f598d95628f68dfd4e1/manifests/source.pp#L75-L83 has code to deal with this